preprocessing/neighbors/n_neighbourhood.py

```diff
@@ -50,21 +42,25 @@
 out_dir = Path(args.out_dir)
 
 spatial_connectivities_file = out_dir / "spatial_connectivities.mtx"
-spatial_distances_file = out_dir / "spatial_distances.mtx"
 
 # Use these filepaths and inputs ...
 coord_file = args.coordinates
 matrix_file = args.matrix
 feature_file = args.features
 observation_file = args.observations
 
-## Custom parameters
-coord_type = args.coord_type
-n_neighs= args.n_neighs
+## Loading parameters from config file
 
 if args.config is not None:
     config_file = args.config
 
+import json
+
+with open(config) as f:
+   parameters = json.load(f)
+
+n_neighs = data["n_neighs"]
+f.close()
```

Comment:
ContextManager will automatically close the file. This line is not needed

Reply:
Removed in all scripts